Formula/latexml.rb

```diff
@@ -14,26 +14,26 @@ class Latexml < Formula
     sha256 "884426eb041a9fa05ba6ebc64c64f4ce76f7c10cab3c5c1b98bcce201831c9d2" => :mavericks
   end
 
-  resource "Image::Size" do
-    url "https://cpan.metacpan.org/authors/id/R/RJ/RJRAY/Image-Size-3.300.tar.gz"
-    mirror "http://search.cpan.org/CPAN/authors/id/R/RJ/RJRAY/Image-Size-3.300.tar.gz"
-    sha256 "53c9b1f86531cde060ee63709d1fda73cabc0cf2d581d29b22b014781b9f026b"
-  end
-
-  resource "Text::Unidecode" do
-    url "http://search.cpan.org/CPAN/authors/id/S/SB/SBURKE/Text-Unidecode-1.27.tar.gz"
-    sha256 "11876a90f0ce858d31203e80d62900383bb642ed8a470c67539b607f2a772d02"
-  end
+  requires_perl = []
+  requires_perl << "with-perl"
+  depends_on "perl"                             # Actually, Imagemagick already installs Perl
+  depends_on "imagemagick" => requires_perl     # Install ImageMagick with PerlMagick
```

Comment:
Also note this is still not allowed. You will need to vendor imagemagick with perl into the formula.